myhpi/search/templates/search/search_result.html

```diff
@@ -3,7 +3,12 @@
 
 <a href="{% pageurl result %}">
     <div class="search-result">
-        <h5>{{ result.title }}</h5>
+        <hgroup>
+            <h5>{{ result.title }}</h5>
+            <time datetime="{{ result.first_published_at|date:"c" }}">
+                {{ result.first_published_at|date:"SHORT_DATE_FORMAT" }}
```

Comment:
this is actually a bit misleading (our bad, the issue states it like you implemented it). We usually only publish our minutes about 5 or 6 days after the actual meeting, after everyone that participated had a chance to review it. Therfore, minutes have an additional field carrying the date of the meeting. My preferred choice would be to only display this date for minutes and nothing for other types of pages, as the date isn't really helpful there IMO